src/app/+display/section/section.component.html

```diff
@@ -1,7 +1,26 @@
-<div class="card-header font-weight-bold text-primary text-capitalize">{{section.name}}</div>
+<div class="card-header font-weight-bold text-primary text-capitalize">
+  {{section.name}}
+  <div class="float-right" *ngIf="isShareable()">
+    <div id="embedDropdown" class="dropdown d-inline-block" ngbDropdown>
```

Comment:
If multiple sections support shareable, this will lead to duplicate ids. Shouldn't this be a class?